specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/vmware.json

```diff
@@ -0,0 +1,1890 @@
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateCloud"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClusterList": {
+      "description": "A paged list of clusters",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AdminCredentials": {
+      "description": "Administrative credentials for accessing vCenter and NSX-T",
+      "properties": {
+        "nsxtUsername": {
+          "description": "NSX-T Manager username",
+          "type": "string",
+          "readOnly": true
+        },
+        "nsxtPassword": {
+          "description": "NSX-T Manager password",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-secret": true
+        },
+        "vcenterUsername": {
+          "description": "vCenter admin username",
+          "type": "string",
+          "readOnly": true
+        },
+        "vcenterPassword": {
+          "description": "vCenter admin password",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-secret": true
+        }
+      }
+    },
+    "Sku": {
+      "description": "The resource model definition representing SKU",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the SKU."
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "HcxEnterpriseSiteList": {
+      "description": "A paged list of HCX Enterprise Sites",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/HcxEnterpriseSite"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "HcxEnterpriseSite": {
+      "description": "An HCX Enterprise Site resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of an HCX Enterprise Site resource",
+          "$ref": "#/definitions/HcxEnterpriseSiteProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "HcxEnterpriseSiteProperties": {
+      "description": "The properties of an HCX Enterprise Site",
+      "properties": {
+        "activationKey": {
```

Comment:
if this is a secret ensure it does not show up in GET/PUT responses and is only visible via a POST action

Reply:
This was also determined not to be a secret in the private review.